content/blog/keda-2.0-release.md

```diff
@@ -0,0 +1,147 @@
+
+With our new **external push scaler**, you can build your own scaler and **trigger scaling actions in KEDA by using a push-model for more reactive autoscaling**, instead of a pull-model with our current pull-based model in our external scaler.
+
+Finally, with our new **Metrics API scaler you can automatically scale on metrics provided through a REST API** without having to build your own scaler. This allows you to make autoscaling decisions based on a metric source that is available in your landscape (for example an in-house API or Microsoft Dynamics CRM API to automate processes).
+
+On top of these new scalers, we’ve made a variety of improvements to the current scalers to optimize our scaling.
+
+### Operations & Extensibility
+
+KEDA now exposes **Liveness and Readiness probes** on both Operator and Metrics server pods, so administrators have a better overview of the state and health of KEDA.
+
+KEDA Metrics Server now **exposes Prometheus metrics about each used scaler**. Currently these metrics work with ScaledObject metrics when the HorizontalPodAutoscaler is active (> 0 replicas). In further releases of KEDA there are plans to extend this capability and serve metrics for ScaledJobs as well.
+
+Developers can now use a **go-client library** that is [exposed](https://github.com/kedacore/keda/tree/main/pkg/generated) to allow easy manipulation of the KEDA API from within applications.
+
+## Migrating to KEDA 2.0
+
+We want it to be super simple to use 2.0 as an existing customer! But what has changed?
+
+We’re making some general changes:
+
+- API namespace for KEDA Custom Resources Definitions (CRD) has changed from `keda.k8s.io` to `keda.sh`
+- Scaling jobs is now done throught `ScaledJob` CRD, instead of `ScaledObject` CRD
+- `ScaledObject` is now using `spec.scaleTargetRef.name`, instead of `spec.scaleTargetRef.deploymentName`
+- `ScaledObject` no longer requires `deploymentName` label _(last couple of v1 releases were already ignoring it)_
+
+In order to provide a more consistent experience across all scalers, we’re introducing improved flexibility & usability of trigger metadata for which we had to do some [cross-scaler breaking changes](https://keda.sh/docs/2.0/migration/#improved-flexibility--usability-of-trigger-metadata) as well as improvements for [Azure Service Bus, Kafka & RabbitMQ scalers](https://keda.sh/docs/2.0/migration/#scalers).
+
+Learn more about how to migrate by using our [migration guide](https://keda.sh/docs/2.0/migration/)!
+
+> **⚠ Running KEDA 1.x & 2.0 side-by-side is not supported.**
+>
+> KEDA comes with a metrics server and Kubernetes only allows you to run one of them in a cluster.
+>
+>*Learn more about how KEDA is architected in [our docs](http://keda.sh/docs/latest/concepts/#architecture).*
+
+## KEDA 💘 Community
```

Comment:
@jeffhollan This bit was added after your review, you might want to take a look at it.